DEPLOYMENT.md

```diff
@@ -2,6 +2,32 @@
 
 Host on Heroku. [Integrated with GitHub](https://devcenter.heroku.com/articles/github-integration).
 
+## Prepare your environment
+
+```
+cd your_projects_folder
+git clone https://github.com/rubytaiwan/rubytw-reboot.git
+heroku login
+heroku git:remote -a rubytw-staging -r staging
+heroku git:remote -a rubytw-production -r production
+git remote -v
+# you will got staging/production remote name now.
+```
+
+## Local development environment
```

Comment:
@puff-tw We already had [one section in CONTRIBUTING.md](https://github.com/rubytaiwan/rubytw-reboot/blob/master/CONTRIBUTING.md#configure-your-local-development-environment), could you move this content there? :blush: